docs/reference/one-sdk.mdx

```diff
@@ -0,0 +1,134 @@
+---
+slug: '/reference/one-sdk'
+---
+
+# OneSDK for Node.js Reference
+
+OneSDK is a universal API client for Node.js. This page describes OneSDK public API and configuration options.
+
+## Important links
+
+- [OneSDK on GitHub](https://github.com/superfaceai/one-sdk-js) for development and reporting issues
+- [OneSDK on npm](https://www.npmjs.com/package/@superfaceai/one-sdk)
+- [Getting Started](/getting-started) tutorial
+
+## Installation
+
+OneSDK is available as Node package under the name `@superfaceai/one-sdk`. For installation in a Node.js run:
+
+```shell
+npm install @superfaceai/one-sdk
+```
+
+## Usage
+
+### Configuration
+
+OneSDK requires a configuration file, by default located in `superface/super.json` path. You can create this file using the [Superface CLI](https://github.com/superfaceai/cli) by installing a profile and configuring a provider:
+
+```shell
+npx @superfaceai/cli install <profileName>
+npx @superfaceai/cli configure <providerName> -p <profileName>
+```
+
+// TODO: tip where I can find them?
```

Comment:
Find what exactly? Providers? At `https://superface.ai/<profileName>` ?

Reply:
Oh, looks like this is an omission from the original README. I will remove that.

```suggestion
```